### PR TITLE
Sureshkodati patch 1

### PR DESCRIPTION
**Pull request by sureshkodati**

### CODE REVIEW

**Comment by sureshkodati** [comment]
Approved

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Java CI with Maven
```
Comment (sureshkodati) [inline .github/workflows/maven.yml:9]: check this line

File: pom.xml [hunk 2]
```diff
@@ -102,6 +102,7 @@
                                  </configuration>
 			</plugin>
 		</plugins>
+		<outputDirectory>${project.basedir}/target</outputDirectory>
```
Comment (sureshkodati) [inline pom.xml:105]: target directory

**Review by TheSolutionArchitect — APPROVED** [review]

**Comment by TheSolutionArchitect** [comment]
@sureshkodati , Thank you for your contribution.

**Pull request merged**